### PR TITLE
Remove repeated sentence.

### PR DESCRIPTION
**Pull request by ansky**
# Changes

Remove repeated sentence.  Minor fix to documentation only.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [ ] Run the code checkers with `make check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,8 +133,6 @@ After installing tkn, create a link as kubectl-tkn
 $ ln -s /usr/local/bin/tkn /usr/local/bin/kubectl-tkn
   ```
 
-`kubectl` will find any binary named `kubectl-*` on your `PATH` and consider it as a plugin
```
Comment (danielhelfand) [inline README.md]: We might want something here though, such as:

Run the following to confirm `tkn` is available as a plugin:

File: README.md [hunk 2]
```diff
@@ -133,8 +133,6 @@ After installing tkn, create a link as kubectl-tkn
...
-
   ```shell
 $ ./kubectl plugin list
```
Comment (danielhelfand) [inline README.md]: This should actually be

```suggestion
$ kubectl plugin list
```

Feel free to edit this too!

File: README.md [hunk 3]
```diff
@@ -133,8 +133,6 @@ After installing tkn, create a link as kubectl-tkn
...
 /usr/local/bin/kubectl-tkn
```
Comment (danielhelfand) [inline README.md]: Maybe separate this line from kubectl plugin list and say: 

You should see the following after running `kubectl plugin list` if `tkn` is available as a plugin:

```suggestion
/usr/local/bin/kubectl-tkn
```

File: README.md (hunk 1 above)
Comment (ansky) [reply README.md]: Thanks - applied these changes.

File: README.md [hunk 4]
```diff
@@ -132,14 +132,16 @@ After installing tkn, create a link as kubectl-tkn
   ```shell
 $ ln -s /usr/local/bin/tkn /usr/local/bin/kubectl-tkn
   ```
-
-`kubectl` will find any binary named `kubectl-*` on your `PATH` and consider it as a plugin
-
+Run the following to confirm tkn is available as a plugin:
```
Comment (chmouel) [inline README.md]: I don't thank this is needed isnt it ?

Comment (danielhelfand) [reply README.md]: It's not technically needed, but I think it makes sense as a way to confirm it's available before running `kubectl-tkn`. Better to know it's set up correctly before running commands with it.

Comment (danielhelfand) [inline README.md]:
```suggestion
Run the following to confirm `tkn` is available as a plugin:
```

File: README.md [hunk 5]
```diff
@@ -132,14 +132,16 @@ After installing tkn, create a link as kubectl-tkn
...
-$ ./kubectl plugin list
-/usr/local/bin/kubectl-tkn
+$ kubectl plugin list
   ```
-
+You should see the following after running kubectl plugin list if tkn is available as a plugin:
+  ```shell
+/usr/local/bin/kubectl-tkn
+```
+If the output above is shown, run kubectl-tkn to see the list of available tkn commands to run.
```
Comment (danielhelfand) [inline README.md]:
```suggestion
If the output above is shown, run `kubectl-tkn` to see the list of available `tkn` commands to run.
```

**Comment by piyush-garg** [comment]
@ansky Can you please squash the commits into one

**Comment by chmouel** [comment]
I am fine with this change but please do the squashing as the git history will look more clearer for us.. here there is an article about it https://github.blog/2016-04-01-squash-your-commits/ we cannot do it oursevles on merge since we are using prow as the tool to merge our commit (which doesnt support squashing)

**Comment by danielhelfand** [comment]
/hold 

Let's wait until commits are squashed

**Comment by ansky** [comment]
Commits have been squashed.

**Comment by chmouel** [comment]
/hold cancel
/lgtm
/approve

Thanks 👍🏼🔥

**Comment by danielhelfand** [comment]
@ansky Thanks so much!

**Pull request merged**